### ISSUE TITLE
  jsx-a11y/label-has-associated-control gives error when for is put 

### ISSUE THREAD

**Issue by TheElegantCoding** [jsx-eslint/eslint-plugin-jsx-a11y#961]
This is my config

`'jsx-a11y/label-has-associated-control': 'error'`

i am not in react i use "for", this make me error in the code only when i put htmlFor works but i cant do that i dont use react
```tsx
return (
  <>
    <label for="email" />
    <input id="email" name="email" type="text" />
  </>
)
```

**Comment by ljharb** [comment]
Most jsx renderers match react, since react invented jsx. It’d be reasonable to add a setting that can rename htmlFor as “for”, i suppose.

**Comment by AesthetiCoder** [comment]
not all jsx renderes match that, i am in solidjs so yeah it will be nice

**Comment by felquis** [comment]
It's missing aria-label attribute in the label element

**Comment by TheElegantCoding** [comment]
wating for this support
